### PR TITLE
[Feature] Adding Analytics dashboard to Netmanager

### PR DESCRIPTION
**Pull request by Baalmart**
- What does this PR do?
This PR moves the analytics dashboard to Netmanager - an effort to having one AirQo platform

- How do I test it out?

> For Linux

`cd netmanager`
`npm install`
`npm run dev-mac` or `npm run prod-mac` or `npm run stage-mac`

> For Windows

`cd netmanager`
`npm install`
`npm run dev-pc` or `npm run prod-pc` or `npm run stage-pc`

- Any considerations?
None

### CODE REVIEW

File: netmanager/package.json [hunk 1]
```diff
@@ -1,106 +1,111 @@
...
-    "collectCoverageFrom": [
-      "src/**/*.js",
-      "!src/index.js"
-    ],
-    "coverageReporters": [
-      "text"
-    ]
-  }
-}
+    "name": "locate",
```
Comment (danielogen) [inline netmanager/package.json:2]: We could change this to "platform" or "netmanager".. whatever works.

**Review by danielogen — APPROVED** [review]
Looks good for the start...improvements need for UI and layout of components

**Review by sserurich — APPROVED** [review]
More improvements on the layout still needed.
image

**Review by lilly-muyama — CHANGES_REQUESTED** [review]
Nodes are not showing on the map.

**Review by Priscilla-AL — COMMENTED** [review]
Failed to run branch

**Review by lilly-muyama — APPROVED** [review]

**Pull request merged**